WooCommerce/Classes/POS/Presentation/Card Present Payments/Reader Messages/PointOfSalePaymentSuccessViewModel.swift

```diff
@@ -22,15 +22,15 @@ private extension PointOfSalePaymentSuccessViewModel {
         )
 
         static let messageCard = NSLocalizedString(
-            "pointOfSale.paymentSuccessful.message.card",
-            value: "A card payment of %1$@ was successfully made",
+            "pointOfSale.paymentSuccessful.message.card.1",
```

Comment:
nice, I'll start using this numbered version when updating string value next time.